src/extension.ts

```diff
@@ -58,12 +58,21 @@ import { startTreeView } from "./treeview";
 import { MetalsFeatures } from "./MetalsFeatures";
 import { MetalsTreeViewReveal, MetalsTreeViews } from "./tree-view-protocol";
 import * as scalaDebugger from "./scalaDebugger";
+import * as jdki from "./jdkInstaller";
```

Comment:
Can we try to use consistent names between the file and imported variable?
```suggestion
import * as jdkInstaller from "./jdkInstaller";
```